src/qilib/configuration_helper/adapters/uhfli_instrument_adapter.py

```diff
@@ -30,9 +30,11 @@ def __init__(self, address: str) -> None:
         self._instrument: ZIUHFLI = ZIUHFLI(self.name, device_ID=address)
 
     def _filter_parameters(self, parameters: PythonJsonStructure) -> PythonJsonStructure:
-        for parameter, values in parameters.items():
+        for _, values in parameters.items():
```

Comment:
Since you're only interested in the dictionary's values, why not iterate over its values? E.g.
```python
for values in parameter.values():
```